docs/user-guide/cli-using-integrating-apiml.md

```diff
@@ -45,15 +48,49 @@ Follow these steps to request a token and log in to API ML:
     Where the token is saved depends on whether you have an existing base profile and where that profile is located. To learn about the precedence Zowe CLI follows with profile configurations, see [How configuration files and profiles work together](../user-guide/cli-using-understand-profiles-configs.md#how-configuration-files-and-profiles-work-together).
     :::
 
-3. Provide a base path and base profile on commands to connect to API ML.
+3. If you do not have a profile for the service, respond to Zowe CLI prompts for connection information to create a profile for the service.
+
+    :::tip
+    - To establish a base path, see instructions for [Zowe V2 profiles](#specifying-a-base-path-with-zowe-v2-profiles) or [Zowe V1 profiles](#specifying-a-base-path-with-zowe-v1-profiles).
+    :::
+
+    If you already have a profile in your configuration for the service you want to connect to, use a text editor to open the applicable configuration file and replace the `port` property with a `basePath` property to enable the use of the API ML.
+
+    A profile with a port number:
+
+    ```json
+        "type": "zosmf",
+        "properties": {
+            // highlight-start
+            "port": 443
+            // highlight-end
+        }
+    ```
+
+    A profile with a base path:
+
+    ```json
+    "type": "zosmf",
+        "properties": {
+              // highlight-start
+            "basePath": "/ibmzosmf/api/v1"
+              // highlight-end
+        }
+    ```
+
 
-    To establish a base path, see instructions for [Zowe V2 profiles](#specifying-a-base-path-with-zowe-v2-profiles) or [Zowe V1 profiles](#specifying-a-base-path-with-zowe-v1-profiles).
 
      If you use the  `--show-token` option with the `login` command, you must manually supply the token on each command using the `--token-value` option. For example:
 
      ```
-     zowe zos-files list data-set "IBMUSER.*" --base-path "ibmzosmf/api/v1" --token-value "123"
+     zowe zos-files list data-set <IBMUSER.*> --token-value <123>
      ```
+    - `<IBMUSER.*>`
+
+        Specifies the data set search criteria
```

Comment:
Wrapping values in angle brackets makes sense for placeholders like `<APIML Host>` and `<APIML Port>` below. In this case `IBMUSER.*` is an example of a real value rather than a placeholder, so wondering if we should say something like the following:
```suggestion
     zowe zos-files list data-set <searchPattern> --token-value <123>
     ```
    - `<searchPattern>`

        Specifies the data set search criteria (e.g., `IBMUSER.*`)
```